presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -175,4 +188,21 @@ private void execute(String sql)
             statement.execute(sql);
         }
     }
+
+    private QueryRunner createQueryRunnerWithLegacyCoercion()
+            throws Exception
+    {
+        Map<String, String> properties = ImmutableMap.<String, String>builder()
+                .put("deprecated.legacy-char-to-varchar-coercion", "true")
```

Comment:
You can make this shorter, one-line, with `ImmutableMap.of("deprecated.legacy-char-to-varchar-coercion", "true")`